### ISSUE TITLE
Cannot create directory '"/mnt/sdcard/.config/qimessaging"

### ISSUE THREAD

**Issue by NeoLuk** [aldebaran/qisdk-tutorials#6]
Hi all, I have cloned the repo and install to my pepper and everything is good.
but the app only shows the layout and without functioning. The full error log is pasted below:

> E/MultiWindowProxy: getServiceInstance failed!
E/qi.path.sdklayout: Cannot create directory '"/mnt/sdcard/.config/qimessaging"' error was: boost::filesystem::create_directories: Permission denied: "/mnt/sdcard/.config"
E/qi.os: Unable to create file: ''>

if someone can help would be great!

**Comment by Adriencerbelaud** [comment]
Hi, same problem here. Can you tell me how you solved this? Thx

**Comment by tcruzcast** [comment]
Hello @NeoLuk @Adriencerbelaud 
Which is your Pepper's version?
This application uses the new SDK for Android called the Qi SDK which requires the latest Pepper version. You may as well to try it on a virtual robot: https://qisdk.softbankrobotics.com/sdk/doc/pepper-sdk/ch1_gettingstarted/running_application.html#testing-on-a-virtual-robot
Cheers!

**Comment by Adriencerbelaud** [comment]
Hi @tcruzcast  and thx, I was testing on a virtual robot. When running "Hello human", speech appears in Dialog view. But nothing appears on the tablet.

Only error in the logcat is :
E/qi.path.sdklayout: Cannot create directory '"/mnt/sdcard/.config/qimessaging"' error was: boost::filesystem::create_directories: Permission denied: "/mnt/sdcard/.config"
E/qi.os: Unable to create file: ''

Edit : made some tests with qisdk-tutorials and everything works... ^^

**Comment by NeoLuk** [comment]
I have found out that this error message will not make the program not functioning at all. It just shows and actually won't harm the program (but if the Permission error can be solved will be so great). 

The robot is not functioning in my case is because of something else.
The thing I have discovered is that pepper using Qi SDK requires robot to "wake up", if the robot is at rest, the robot focus will never be gained.
So remember to "wake up" your pepper!!
